docs/faq.rst

```diff
@@ -19,6 +19,17 @@ However, during development of a new model it will be necessary to reinitialize.
 
    2. Restart the clients. 
 
+Q: Can I skip fetching the remote package and instead use a local folder when developing the compute package
```

Comment:
I think I removed this Q in current  master (since --remote False flag currently not working)